Dynatrace-Environment-Metric/overrides.json

```diff
@@ -0,0 +1,16 @@
+{
+  "CREATE": {
+    "/Id": "custom:CFNPublish",
+    "/DisplayName": "CFN Publishing",
+    "/Unit": "Count (count)",
+    "/Dimensions": ["nic"],
+    "/Types": ["F5-Firewall"]
+  },
+  "UPDATE": {
+    "/Id": "custom:CFNPublish",
+    "/DisplayName": "CFN Publishing",
```

Comment:
The update is identical to the create. This may be an eventual consistency issue but I did not investigate any further